src/main/java/io/openliberty/tools/intellij/LibertyModuleNode.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2020, 2023 IBM Corporation.
+ * Copyright (c) 2020, 2024 IBM Corporation.
```

Comment:
why does this file only have a copyright header update?

Reply:
@TrevCraw  Updated all the other header files as well.

Reply:
@aparnamichael The previous update to this file was in 2023. I am also puzzled why this file is updated in this PR if the only change is to the header. We update the copyrights when the content has changed and it doesn't appear that has happened here.

Reply:
oh, I accidentally changed the year in header file. I am reverting the change.